docs/import/add_disaster.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Add New Disaster</title>
+  <!-- TODO(janakr): Switch back to compiled version in production. -->
+  <!-- TODO(janakr): This is pinned to a particular version, but we should be targeting the live
+       version, whatever it is. -->
+  <script src="https://rawcdn.githack.com/google/earthengine-api/3bb86bfc4f3d9eed98220f3d225b414982915b86/javascript/build/ee_api_js_debug.js"></script>
+  <!-- TODO(janakr): download this script to lib. -->
+  <script src="https://apis.google.com/js/api.js"></script>
+
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-app.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/6.3.3/firebase-firestore.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/7.2.1/firebase-auth.js"></script>
+  <!-- Load jQuery, a 3rd-party library. -->
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+  <script src="https://apis.google.com/js/api.js"></script>
+  <script type="module" src="./add_disaster_startup.js"></script>
+</head>
+<body>
+
+<form action="" id="disaster-picker">
+  <label for="disaster">Disaster: </label>
```

Comment:
it might look a bit better to have the current disasters and "Create New Disaster" on separate lines. It also might be nice to gray out or otherwise indicate that the apparently currently selected disaster isn't actually relevant during disaster creation. Maybe making it invisible and then having a "cancel" button in the form. Right now, if you happen to click on the drop-down disaster list you'll lose any work you might have in progress on disaster creation. Maybe add a TODO?

Reply:
I actually like having create new disaster on the same line, I did that on purpose because i think it's nice and tidy up there. I don't think it should be below (covered by the drop down) and above also seems slightly weird because I think the picker should be front and center. 

Agree though that it's confusing to have any old disaster in the picker while creating a new one. Added a todo to add_disaster_startup. to fix in a follow up.